tutorials/server/enable-the-livepatch-service/enable-the-livepatch-service.md

```diff
@@ -11,12 +11,12 @@ author: Konrad Krawiec <konrad11901@gmail.com>
 
 ---
 
-# Enable the Canonical Livepatch service
+# Apply kernel patches without rebooting
 
 ## Overview
 Duration: 1:00
 
-[Canonical Livepatch service](https://www.ubuntu.com/server/livepatch) applies critical kernel security patches without rebooting Ubuntu.
+The [Canonical Livepatch Service](https://www.ubuntu.com/server/livepatch) applies critical kernel security patches without rebooting Ubuntu. This is especially useful on production environments, databases, and generally systems that shouldn't be disrupted.
```

Comment:
I would say:

This is especially useful on production environments and services where any downtime could be disruptive